src/main/java/controller/UserController.java

```diff
@@ -4,57 +4,66 @@
 import model.User;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import request.HttpRequest;
 import util.RequestParser;
 
-import java.util.*;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Objects;
 import java.util.stream.Collectors;
 
 public class UserController {
 
-        private Logger log = LoggerFactory.getLogger(getClass());
-        // 인스턴스 변수
-        private static UserController instance;
+    private Logger log = LoggerFactory.getLogger(getClass());
+    // 인스턴스 변수
+    private static UserController instance;
 
-        // private 생성자
-        private UserController() {
+    // private 생성자
+    private UserController() {
 
-        }
+    }
 
-        // 인스턴스 반환 메소드
-        public static UserController getInstance() {
-            if (instance == null) {
-                synchronized (UserController.class) {
-                    if (instance == null) {
-                        instance = new UserController();
-                    }
+    // 인스턴스 반환 메소드
+    public static UserController getInstance() {
+        if (instance == null) {
+            synchronized (UserController.class) {
+                if (instance == null) {
+                    instance = new UserController();
                 }
             }
-            return instance;
         }
+        return instance;
+    }
 
-        public String mapToFunctions(String httpMethod, String resourceUrl) {
+    public String mapToFunctions(HttpRequest httpRequest) {
 
-            String data = null;
+        String data = null;
+        String httpMethod = httpRequest.getValueByNameInRequestLine("httpMethod");
```

Comment:
```suggestion
        String httpMethod = httpRequest.getMethod();
```
HTTPReqeust 객체라면 반드시 있어야 하는 필드들은 별도의 멤버로 가지고 있어도 좋을 것 같아요.